client/client.go

```diff
@@ -300,21 +300,23 @@ func (c *CvpClient) makeRequest(reqType string, url string, params *url.Values,
 	data interface{}) ([]byte, error) {
 	var err error
 	var resp *resty.Response
+	var formatedParams map[string]string
```

Comment:
Change variable name to formattedParams.

Reply:
Fixed.